### ISSUE TITLE
Set up "Discussions" for this and the lab repos?

### ISSUE THREAD

**Issue by NicMcPhee** [UMM-CSci-3601/3601-iteration-template#854]
We should consider setting up the new-ish [GitHub Discussions tool](https://github.com/UMM-CSci-3601/3601-iteration-template/discussions) as a way for a potentially broader audience (including the students in the course) to comment on and ask questions about the repos in this organization.

**Comment by NicMcPhee** [comment]
Just for fun I "turned on" Discussions for this repo to see what that looks like. #872

**Comment by floogulinc** [comment]
Note that you can have discussions at the repo level or for an entire organization. Organization wide discussions would be particularly useful for this course as they wouldn't be tied to a single teams's repo so they could be used for all kinds of general things in the course, for example database structuring standards.

**Comment by floogulinc** [comment]
In fact we should probably use them at the org level here instead of in the template repos.

**Comment by NicMcPhee** [comment]
I had looked into this and somehow convinced myself that orgs couldn't have
discussions. They can, but those have to be attached to a specific repo, so
you're really just making a particular repo's discussion into the org-level
discussion.

I just made the iteration template into "that repo" so we could see what
that looks like. I wonder if we want a separate "discussion" repo that
hosts this discussion instead of having it attached to something like the
template, which could go away altogether if, e.g., we move to a monorepo,
or end up changing our front-end tooling, etc., etc.

**Comment by floogulinc** [comment]
Yeah that could be a good idea, I could also see making one for each semester org at the begining to hold the general discussions and any shared assets could be stored in the repo too.

**Comment by NicMcPhee** [comment]
If we like this structure, then I may want to start moving "general" issues to that repo instead of having them in a single lab (or template) repo when they apply to multiple things.

**Comment by floogulinc** [comment]
Don't forget GitHub has a "transfer issue" and "transfer discussion" button you can use to automatically move them between repos instead of remaking them.

![Screenshot_20220425-153249](https://user-images.githubusercontent.com/1300395/165170136-06aa2f5e-9a12-4995-98ee-ab34f0a0f3a3.png)

**Comment by NicMcPhee** [comment]
Doh! Is totally missed that!
